doc/tutorial/01_install.md

```diff
@@ -143,11 +149,16 @@ Add the master key and create vault
 pgmoneta-admin master-key
 pgmoneta-admin -f pgmoneta_users.conf -U repl -P secretpassword add-user
 ```
-
 You have to choose a password for the master key - remember it !
 
+If you see an error saying `error while loading shared libraries: libpgmoneta.so.0: cannot open shared object` running 
+the above command. you may need to locate where your `libpgmoneta.so.0` is. It could be in `/usr/local/lib` or `/usr/local/lib64`
+depending on your environment. Add the corresponding directory into `/etc/ld.so.conf`, or alternatively, create a file
+called `pgmoneta_shared_library.conf` under `/etc/ld.so.conf.d/`, and add your directory into it. Remember to run `ldconfig` to make the change effective
+
 Create the `pgmoneta.conf` configuration
 
+Again, use `/usr/bin/` as `pgsql_dir` instead if you used dnf to install postgres
```

Comment:
Did you mean `/usr/pgsql-10/bin/` here ?

Reply:
Yes, I elaborate on this in the newest commit, unless you mean I'm not supposed to replace it?